src/App.js

```diff
@@ -39,7 +39,11 @@ class App extends Component {
         value={{ score: this.state.score, updateScore: this.updateScore }}
       >
         <div className="App">
-          <Sidebar books={books} score={this.state.score} />
+          <ScoreContext.Consumer>
```

Comment:
This is defintely the right idea. however, we don't actually need to use `<ScoreContext.Consumer />` here because we have direct access to `score` and `updateScore` in `App.js`. We can just pass `score` and `updateScore` directly as props.